<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>349</width>
    <height>236</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DI Pattern Match Interrupt - Configuration</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QComboBox" name="cmbDevice">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblDevice">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Device:</string>
   </property>
   <property name="buddy">
    <cstring>cmbDevice</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblDIPort">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>DI port:</string>
   </property>
   <property name="buddy">
    <cstring>cmbDIport</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbDIport">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblEnChanel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Enabled channels:</string>
   </property>
   <property name="buddy">
    <cstring>cmbDevice</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="lblMatValue">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>193</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Matched value:</string>
   </property>
   <property name="buddy">
    <cstring>cmbDevice</cstring>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox1">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>155</y>
     <width>108</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>108</width>
     <height>31</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn07">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonUp.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn06">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonUp.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn05">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonUp.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn04">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonUp.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox2">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>156</y>
     <width>108</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>108</width>
     <height>31</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn03">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonUp.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn02">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonUp.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn01">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonUp.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn00">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonUp.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox3">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>216</y>
     <width>108</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>108</width>
     <height>31</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn17">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonDown.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn16">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonDown.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn15">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonDown.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn14">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonDown.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox4">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>216</y>
     <width>108</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>108</width>
     <height>31</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn13">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonDown.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn12">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonDown.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn11">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonDown.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn10">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>2</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/DIPatternMatchInterrupt/Resources/ButtonDown.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="txtenableChan">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtpmValue">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string>FF</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Profile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Profile:</string>
   </property>
   <property name="buddy">
    <cstring>cmbDevice</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtProfilePath">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowse">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cmbDevice</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="dipatternmatchinterrupt.qrc"/>
 </resources>
 <connections/>
</ui>
